lib/get-group-tangle.js

```diff
@@ -12,29 +12,31 @@ const { isIdentityGroupSSBURI } = require('ssb-uri2')
 
 const strategy = new Strategy({})
 
-module.exports = function GetGroupTangle(server) {
-  const getUpdates = GetUpdates(server)
+/** `server` is the ssb server you're using. `tangle` is the name of the tangle in the group you're looking for, e.g. "group" or "member" */
+module.exports = function GetGroupTangle(server, tangle) {
+  const getUpdates = GetUpdates(server, tangle)
 
-  return function getGroupTangle(groupId, cb) {
+  return function getTangle(groupId, cb) {
```

Comment:
ah yeah maybe makes sense. i was thinking it still made some sense since i don't think it lets you find tangles outside of groups, but i guess since the function is limited to inside this library it's ok